docs/custom_tagging.rst

```diff
@@ -12,6 +12,8 @@ want to store additional data about a tag, such as whether it is official.  In
 these cases ``django-taggit`` makes it easy to substitute your own through
 model, or ``Tag`` model.
 
+Note: You will need to remove 'taggit' from `settings.py` if you do not want to create the default tables.
```

Comment:
can you be more specific? the sentence needs more detail and context.

Reply:
@auvipy I've updated the Note. Let me know if it needs more detail!